Dockerfile

```diff
@@ -3,10 +3,11 @@ FROM python:3.8-slim
 RUN mkdir -p /app
 WORKDIR /app
 
-COPY requirements.txt requirements.txt
-RUN pip install -r requirements.txt
+RUN apt-get update && apt-get install -y git
```

Comment:
if we derive from `python:3.8` we get git for free; it's probably not an important difference